ingestion-beam/bin/build-template

```diff
@@ -65,6 +65,12 @@ elif [[ $JOB_TYPE =~ ^republisher_.*$ ]] ; then
 --outputFileFormat=json ${SINK_OPTS} ${REPUBLISHER_OPTS:?}"
   ERROR_OUTPUT_OPTS="--errorOutputType=file \
 --errorOutputNumShards=${ERROR_NUM_SHARDS}"
+elif [[ $JOB_TYPE =~ ^contextualservices$ ]] ; then
```

Comment:
@jklukas Can you take a look at this?  Do job specific options like `allowedDocTypes` belong here?

Reply:
We only need to specify compile-time options here, since those are necessary at the point we're building a template.

The new options specific to ContextualServicesReporter are all runtime options (using the ValueProvider interface), so those don't need to be specified here. Instead, they'll get specified in deploy-time ops logic.